R/fct_appdata.R

```diff
@@ -267,9 +267,15 @@ get_appdata <-  function(
       "item_group consists of multipe elements which is not allowed: ", 
       item_group_x
     )
+    form_type_x <- unique(with(var_levels, form_type[item_group == item_group_x]))
+    if(length(form_type_x) != 1) stop(
+      "form_type consists of multipe elements which is not allowed: ", 
+      form_type_x
+    )
+    class(x) <- c(form_type_x, class(x))
     tableclass <- simplify_string(item_group_x)
     if(tableclass %in% tableclasses){
-      class(x) <- c(tableclass, class(x)) 
+      class(x) <- unique(c(tableclass, class(x)))
```

Comment:
I'm not against this, but I was just wondering if there was a specific reason to wrap this in `unique`? Did you encounter a problem without it?

Reply:
Just to be tidy. Otherwise there are/can be duplicates in the class of an object.